### PR TITLE
[[TEST]] Fixing copy-paste bugs in tests

### PR DESCRIPTION
**Pull request by jshaptic**
Noticed several copy-paste errors in test scripts. Modified names of some functions and restored, what was intended.

### CODE REVIEW

File: tests/unit/parser.js [hunk 1]
```diff
@@ -4404,47 +4404,47 @@ exports["for of as legacy JS"] = function (test) {
...
-    .addError(15, 6, "Invalid for-of loop left-hand-side: initializer is forbidden.")
-    .addError(15, 6, "'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).")
-    .addError(16, 17, "'for of' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).")
-    .addError(16, 13, "Invalid for-of loop left-hand-side: more than one ForBinding.")
-    .addError(16, 6, "'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).")
-    .addError(17, 25, "'for of' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).")
-    .addError(17, 17, "Invalid for-of loop left-hand-side: initializer is forbidden.")
-    .addError(17, 17, "Invalid for-of loop left-hand-side: more than one ForBinding.")
-    .addError(17, 6, "'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).")
-    .test(code, {undef: true, predef: ["print"]}); // es5
```
Comment (rwaldron) [inline tests/unit/parser.js]: These changes are invalid. The expected character number was added https://github.com/jshint/jshint/commit/e0d575f2803ff9d5891007f2c22c3953df4a21fe

**Comment by rwaldron** [comment]
@jugglinmike these changes look good to me, can you give this a review? Thanks!

**Comment by rwaldron** [comment]
@jshaptic you'll still need to rebase this before it can land.

**Comment by jshaptic** [comment]
Rebase is executed.

**Comment by jugglinmike** [comment]
Looking good! Squashed and merged to `master` at commit 68be275. Thanks :)

**Pull request closed without merging**